doc/sphinx/reference/vmod.rst

```diff
@@ -264,6 +264,19 @@ managing instances, in particular their memory management. As the
 lifetime of object instances is the vcl, they will usually be
 allocated from the heap.
 
+Functions and Methods scope restriction
+---------------------------------------
+
+The ``$Restrict`` stanza offers a way to limit the scope of the preceding vmod function
+or method, so that they can only be called from restricted vcl call sites.
+It must only appear after a ``$Method`` or ``$Function`` and has the following syntax::
+
+    $Restrict scope1 [scope2 ...]
+
+Possible scope values are:
+backend, client, housekeeping, recv, pipe, pass, hash, purge, miss, hit,
```

Comment:
why not name the scopes `vcl_*` except for the "meta scopes" `backend, client, housekeeping`

Reply:
Thanks for the review. I added the other suggested change. Regarding the naming, I don't have an opinion, we should probably discuss it with the others during bugwash.

Reply:
Changed the names of scopes as agreed during bugwash to vcl_* (except for backend client and housekeeping)